js/foundation.accordion.js

```diff
@@ -153,6 +153,9 @@ class Accordion {
    * @function
    */
   toggle($target) {
+    if ($target.closest('.accordion').is('[disabled]')) {
```

Comment:
Since we're determining behavior, this should probably look for the data attribute not the class (we use classes for display/styles, data attributes for behavior/js)... so something like
```
if ($target.closest('[data-accordion]').is('[disabled]')) {
```